master/pkg/tasks/copy.go

```diff
@@ -40,6 +40,7 @@ func harnessArchive(harnessPath string, aug *model.AgentUserGroup) cproto.RunArc
 		panic(errors.Wrapf(err, "error finding Python wheel files for version %s in path: %s",
 			version.Version, harnessPath))
 	}
+
```

Comment:
stray change?

Reply:
My editor adds newlines to the ends of files by default if there isn't one there already, which we should be doing anyway. I was curious, though, so I whipped up a quick one-liner to count instances of files with newlines at the end vs. files without (after a `make clean`): `find . -type f -name "*" -print0 | xargs -0 -P 16 -I{} -S1024 /bin/bash -c 'if [[ $(tail -c1 {} 2>/dev/null | wc -l) -gt 0 ]]; then echo 'newline'; else echo 'none'; fi' ';' | sort | uniq -c` It's ugly, but it seems to work. I get:

```
   5351 newline
   1327 none
```